src/main/resources/templates/fragments.html

```diff
@@ -16,31 +16,48 @@
 
 <body>
 
-    <nav th:fragment="page-header" class="navbar navbar-expand-md bg-success navbar-dark navbar-fixed-top">
-        <!-- Logo -->
-        <img th:src="@{/images/logo.jpg}" alt="Logo" class="logo">
```

Comment:
you'll see a new logo.jpg got pushed, i recropped the image to look a lot cleaner.